docs/guides/src/main/server/containers.adoc

```diff
@@ -4,18 +4,18 @@
 <#import "/templates/links.adoc" as links>
 
 <@tmpl.guide
-title="Running in a container"
+title="Running Keycloak in a container"
 summary="Learn how to run Keycloak from a container image"
 priority=20
 includedOptions="db db-url db-username db-password features hostname https-key-store-file https-key-store-password metrics-enabled">
 
-Keycloak is handling containerized environments like Kubernetes or OpenShift as first-class-citizens. In this guide you'll learn how to run and optimize the Keycloak container image to have the best experience running a Keycloak container.
+Keycloak handles containerized environments such as Kubernetes or OpenShift as first-class citizens. This guide describes how to optimize and run the Keycloak container image to provide the best experience running a Keycloak container.
 
-== Create an optimized container image
-To get the best startup experience for your Keycloak container, we recommend building an optimized container image by running the `build` step explicitly before starting:
+== Creating an optimized container image
+For the best start up of your Keycloak container, build an optimized container image by running the `build` step immediately before starting.
```

Comment:
Process-wise, this doesn't have to happen immediately before starting, it just has to happen any time before you start it. Suggestion: remove "immediately" completely.